content.js

```diff
@@ -219,7 +219,7 @@ function resetValues() {
   //reset sponsor times
   sponsorTimes = null;
   UUIDs = null;
-  sponsorVideoID = id;
+  sponsorVideoID = (videoID === undefined) ? false : videoID;
```

Comment:
Change this line once videoID is set to false by default.

Reply:
Done